packages/core/src/payment/strategies/stripe-upe/stripe-upe-payment-strategy.ts

```diff
@@ -17,6 +17,7 @@ import PaymentStrategy from '../payment-strategy';
 import formatLocale from './format-locale';
 import { AddressOptions, StripeConfirmPaymentData, StripeElement, StripeElements, StripeElementType, StripeError, StripePaymentMethodType, StripeStringConstants, StripeUPEAppearanceOptions, StripeUPEClient } from './stripe-upe';
 import StripeUPEScriptLoader from './stripe-upe-script-loader';
+import {StripeUPEPaymentInitializeOptions} from './index';
```

Comment:
```suggestion
import { StripeUPEPaymentInitializeOptions } from './index';
```

Reply:
thank you for the comment, already applied the change